src/template/wasm2c.declarations.c

```diff
@@ -41,7 +41,7 @@ static inline bool func_types_eq(const wasm_rt_func_type_t a,
     TRAP(OOB);
 #endif
 
-#if WASM_RT_MEMCHECK_SIGNAL_HANDLER
+#if WASM_RT_MEMORY_CHECK_GUARD
```

Comment:
Somehow `WASM_RT_MEMORY_CHECK_GUARD` doesn't scan in the way that I would like it to.    Not a huge deal, of course.   How about `WASM_RT_USE_GUARD_PAGES`?